config/initializers/secure_headers.rb

```diff
@@ -6,11 +6,13 @@
   config.x_download_options = 'noopen'
   config.x_permitted_cross_domain_policies = 'none'
 
+  # On the development environment only we allow 'https://*' to hop over to identity-pki 
```

Comment:
Comment looks good to me, but looks like some trailing whitespace upset code climate